src/lib/util.ts

```diff
@@ -32,4 +32,24 @@ export default class Util {
         return "Unknown network"
     }
   }
+
+  /**
+   * Subscribes to pending transactions for a specific operation.
+   * Unsubscribes automatically.
+   * @param topic Arc.js TransactionService topic
+   * @param key A unique `key` to look for in `txEventInfo.options`
+   * @param cb A callback to execute on each pending transaction.
+   */
+  public static onPendingTransactions(topic: string, key: symbol, cb: (txCount: number) => any) {
```

Comment:
the tslint rules should require that function return types be declared

Reply:
I don't think that's always wanted. Though, the return type for this specific function was declared in https://github.com/daostack/alchemy/pull/61/commits/366f5bd5e619d50fa3ea85893029b5ec18f705ad

Reply:
@dev-matan-tsuberi  Just curious:  what are the cases where a return type declaration would not be wanted?

Reply:
For example standard `React` functions like `render` etc... where you know what your'e doing and the types just make things more cluttered.